src/main.rs

```diff
@@ -307,9 +306,11 @@ fn build_snapshot_settings_window(parent: &gtk::Window) {
         if let Some(parent) = s.parent() {
             if !parent.exists() {
                 if let Err(e) = create_dir_all(parent) {
-                    eprintln!("Error when trying to build settings folder '{}': {:?}",
```

Comment:
You're part of those kind of indenters. I see. :p

Reply:
rustfmt :) I don't think about these things